default/scripting/policies/INDOCTRINATION.focs.txt

```diff
@@ -10,25 +10,45 @@ Policy
         [[SPECIES_LIKES_OR_DISLIKES_POLICY_STABILITY_EFFECTS]]
 
         // makes all planets more stable over time at the cost of influence
+        // First variant: Current population was colonized.
+        // Planet was never conquered or it was conquered, emptied and re-colonised.
+        // Bonus is based on only how long indoctrination has been adapted.
         EffectsGroup
         scope = And [
             Planet
             OwnedBy empire = Source.Owner
             Species
+            Or [(LocalCandidate.TurnsSinceLastConquered == 0)
+                (LocalCandidate.TurnsSinceLastConquered > LocalCandidate.TurnsSinceColonization)]
         ]
         effects = [
             SetTargetHappiness value = Value + 
                 min((NamedReal name = "PLC_INDOCTRINATION_STRENGTH_MAX" value = 10.0),
-                    (Target.TurnsSinceLastConquered != 0 ?
-                     min(Target.TurnsSinceLastConquered,
-                         TurnsSincePolicyAdopted empire = Source.Owner name = ThisPolicy) :
-                     TurnsSincePolicyAdopted empire = Source.Owner name = ThisPolicy) *
-                     (NamedReal name = "PLC_INDOCTRINATION_PER_TURN" value = 0.25))
+                    (TurnsSincePolicyAdopted empire = Source.Owner name = ThisPolicy) *
+                    (NamedReal name = "PLC_INDOCTRINATION_PER_TURN" value = 0.25))
+            SetTargetInfluence value = Value - 1.0
+        ]
+        // Second variant: Current population was conquered.
+        // Bonus is additonally limited to turns since conquered.
+        EffectsGroup
+        scope = And [
+            Planet
+            OwnedBy empire = Source.Owner
+            Species
+            // Note: TurnsSinceColonization cannot be equal to TurnsSinceLastConquered
+            And [(LocalCandidate.TurnsSinceLastConquered != 0)
```

Comment:
what is the nested `And` for?

Reply:
See comments before the two effect groups.
Perhaps I should move the comment into the scope, right in front of the Or/And to make it clearer?

Reply:
The `Or` on line 21 is fine, but here you've essentially got `And [ C1 And [ C2 C3 ] ]` which is the same logic as `And [ C1 C2 C3 ]`

Reply:
Ah, now I see. Well, I basically reverted the Or from the first condition, overlooking the fact that I am within an And condition. Will update it.